terra-env/variables.tf

```diff
@@ -157,3 +158,11 @@ variable "janitor_google_folder_id" {
   default     = ""
   description = "The folder ID in which Janitor has permission to cleanup resources."
 }
+
+# Terra RBS Vars
+#
+variable "rbs_google_folder_id" {
+  type        = string
+  default     = ""
```

Comment:
I don't know that this should have an empty string default. If we want by default for it to not create folder permissions, null is better. Otherwise, no default will remind us to give it the right folder permissions.

Reply:
Done